core/wiki/macros/tag-picker.tid

```diff
@@ -36,142 +37,131 @@ second-search-filter: [tags[]is[system]search:title<userInput>sort[]]
 
 \define clear-tags-actions()
```

Comment:
It seems this one did escape ;) It's still a `\define`

Reply:
Oops, good catch!